corehq/apps/data_interfaces/const.py

```diff
@@ -1,4 +1,4 @@
 CRITERIA_OPERATOR_CHOICES = [
-    ('ALL', '(All) The rule will run when ALL of the criteria are met.'),
-    ('ANY', '(Any) The rule will run when ANY of the criteria are met.')
+    ('ALL', 'ALL of the criteria are met'),
```

Comment:
these need translations

Reply:
Thanks for the catch, Jenny. Doesn't look like the SelectToggle widget is set up to handle uggettext_lazy with form choices:

```
File "/Users/addisondunn/Code/commcare-hq/corehq/apps/hqwebapp/widgets.py", line 160, in render
Open an interactive python shell in this frameoptions=html_attr(json.dumps([{'id': c[0], 'text': c[1]} for c in self.choices])))
```

So converting back to the old select list for now

Reply:
That SelectToggle uses `gettext` instead of `gettext_lazy`. Specifically using `gettext_lazy` throws the error above.

I'd use regular `gettext` but I thought we wanted to use `uggettext_lazy` in forms because form code is only run upon Django startup?

Reply:
Yeah I was going to deem changing the `SelectToggle` code out of scope but since I have time today maybe I'll try. Then maybe update the usage of `gettext` to `gettext_lazy`.

Reply:
https://github.com/dimagi/commcare-hq/pull/31561